server/job.go

```diff
@@ -155,14 +156,41 @@ func (c *context) startJob(params map[string]string, startJob lib.StartJob) (*re
 		orchestrationEnv["BZK_CRYPTO_KEYFILE"] = fmt.Sprintf("%s/crypto-key", buildFolder)
 	}
 
+	orchestrationVolumes := []string{
+		fmt.Sprintf("%s:/bazooka", buildFolder),
+		fmt.Sprintf("%s:/var/run/docker.sock", c.Env[BazookaEnvDockerSock]),
+	}
+
+	reuseScmCheckout := project.Config["bzk.scm.reuse"] == "true"
```

Comment:
See my comment here https://github.com/bazooka-ci/bazooka/pull/168#issuecomment-95728198

The inserted value of `bzk.scm.reuse`is `"true"`and not `true` if you set its value with the cli.